web_src/js/index.js

```diff
@@ -698,6 +785,14 @@ function initIssueComments() {
 }
 
 async function initRepository() {
+  $('body').keyup((e) => {
```

Comment:
Not a fan of such a broad key handler, please use a more specific selector.

Reply:
@silverwind it's a global shortcut so not sure how to make this work without body. However I moved it to project.js and only listen to the event if the project element is detected.

Reply:
Hmm, can you put the event listener to something else than body or alternatively use a delegated listener like `$(document).on('keyup', '.parent', () => {})`?